addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/discovery/SamsungTvDiscoveryParticipant.java

```diff
@@ -35,11 +38,21 @@
  * results of searched UPnP devices
  *
  * @author Pauli Anttila - Initial contribution
+ * @author Arjan Mels - Changed to upnp.UpnpDiscoveryParticipant
  */
 @Component(immediate = true)
 public class SamsungTvDiscoveryParticipant implements UpnpDiscoveryParticipant {
     private final Logger logger = LoggerFactory.getLogger(SamsungTvDiscoveryParticipant.class);
 
+    /**
+     * Called at the service activation.
+     *
+     * @param componentContext
+     */
+    @Activate
+    protected void activate(ComponentContext componentContext) {
+    }
```

Comment:
Does this empty method have a reason? If so, please add a comment for it. Otherwise it can be removed.